src/EditorFeatures/TestUtilities/Classification/FormattedClassifications.Json.cs

```diff
@@ -0,0 +1,48 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+using Microsoft.CodeAnalysis.Classification;
+
+namespace Microsoft.CodeAnalysis.Editor.UnitTests.Classification
+{
+    public static partial class FormattedClassifications
```

Comment:
This feels ripe for a source generator...

Reply:
yup!